draft-ietf-quic-load-balancers.md

```diff
@@ -450,45 +450,47 @@ length, splitting an odd octet in half if necessary. For example,
 0x7040b81b55ccf3 would split into a left_0 of 0x7040b81 and right_0 of
 0xb55ccf3.
 
-3. Encrypt the result of expand_left(left_0) to obtain a ciphertext.
+3. Encrypt the result of expand_left(left_0, index)) to obtain a ciphertext,
+where 'index' is 1 plus the total length of the resulting connection ID
+excluding the first octet, cid_len.
 
 4. XOR the least significant bits of the ciphertext with right_0 to form
 right_1.
 
     Thus steps 3 and 4 can be expressed as
     ```
     right_1 = right_0 ^ truncate_right(
-                            AES_ECB(key, expand_left(left_0, 0x01)),
+                            AES_ECB(key, expand_left(left_0, 1 + cid_len)),
                             len(right_0))
```

Comment:
"1+cid_len" is not ideal, especially if we have to bump the count to 12. I would rather have something "len on 8 bits + rank on 4 bits"

Reply:
I was envisioning all this fitting in one byte (which was not clear in the text), but I agree parsing the space is a better result.